src/Sparrow/ByteString.cs

```diff
@@ -284,14 +284,14 @@ public void CopyTo(int from, byte[] dest, int offset, int count)
             }
         }
 
-        public override string ToString()
```

Comment:
Huh? Why aren't you overriding tostring?

Reply:
Because I copy and pasted, and made a mistake...